server/repository/src/migrations/v1_08_00/assets/asset.rs

```diff
@@ -41,5 +41,14 @@ pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
         "#,
     )?;
 
+    if cfg!(feature = "postgres") {
+        sql!(
+            connection,
+            r#"
+            ALTER TYPE context_type ADD VALUE 'ASSET' AFTER 'REPACK';
```

Comment:
```suggestion
            ALTER TYPE context_type ADD VALUE 'ASSET' BEFORE 'INBOUND_SHIPMENT';
```
Nit: If you want the order consistent between the postgres and rust?

Reply:
cool thanks. wasn't sure if the order mattered, but I'd prefer it alphabetical